docs/guide/getting-started.md

```diff
@@ -99,6 +99,9 @@ end
 
 _Note: Content can't be passed to a component via a block in controllers. Instead, use `with_content`. In versions of Rails < 6.1, rendering a ViewComponent from a controller doesn't include the layout._
 
+_Note: If you are working with [turbo-rails](https://github.com/hotwired/turbo-rails), don't forget to set the
```

Comment:
What about adding a code example just below this? It will make this clearer I think.

Reply:
Agreeed.

Reply:
@Spone @joelhawksley Sure, I just added a code example, can you check again?